analysis_options.yaml

```diff
@@ -1,7 +1,8 @@
 analyzer:
```

Comment:
Why generally is this file different from https://github.com/flutter/flutter/blob/master/analysis_options.yaml#L24 ?

Reply:
I think just historically, and to avoid copying and pasting copies of the flutter analysis options. I updated this to try using an analysis options `include:` directive; we'll see if travis is happy with that.

Reply:
wow, i didn't know include: was a thing. If it is we may be able to simplify a number of other files.

Reply:
Yup, developed I think for use by Flutter, but not yet rolled out in use. We'll give it a go here and see how it works. /cc @danrubel